src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -589,4 +604,23 @@ public void onAddToDownloadList() {
 		view.setAddToDownloadList(newAddToDownloadList);
 		newAddToDownloadList.configure(entityBundle.getEntity().getId(), currentQuery);
 	}
+
+	@Override
+	public DatasetEditorProps getItemsEditorProps() {
+		DatasetEditorProps props =
+				DatasetEditorProps.create(
+						entityBundle.getEntity().getId(),
+						() -> closeItemsEditor(),
+						() -> closeItemsEditor()
+				);
+		return props;
+	}
+
+	public void closeItemsEditor() {
+		isShowingItemsEditor = false;
+		actionMenu.setActionVisible(Action.EDIT_DATASET_ITEMS, true);
+		view.setItemsEditorVisible(false);
+		view.setQueryResultsVisible(true);
+		reconfigureState();

```

Comment:
The only thing that *needs* to happen is calling `checkState` again, but I think it's safer to just capture the three methods called in the configure and call all of them.